### ISSUE TITLE
azure-functions-maven-plugin does not support slots

### ISSUE THREAD

**Issue by IrimieBogdan** [microsoft/azure-maven-plugins#581]
### Plugin name and version
name: `azure-functions-maven-plugin`
version: `1.2.2`

### Plugin configuration in your `pom.xml`
```
            <plugin>
                <groupId>com.microsoft.azure</groupId>
                <artifactId>azure-functions-maven-plugin</artifactId>
                <configuration>
                    <resourceGroup>${functionResourceGroup}</resourceGroup>
                    <appName>${functionAppName}</appName>
                    <region>${functionAppRegion}</region>
                    <appSettings>
                        <!-- Run Azure Function from package file by default -->
                        <property>
                            <name>WEBSITE_RUN_FROM_PACKAGE</name>
                            <value>1</value>
                        </property>
                        <property>
                            <name>FUNCTIONS_EXTENSION_VERSION</name>
                            <value>~2</value>
                        </property>
                        <property>
                            <name>FUNCTIONS_WORKER_RUNTIME</name>
                            <value>java</value>
                        </property>
                    </appSettings>
                </configuration>
                <executions>
                    <execution>
                        <id>package-functions</id>
                        <goals>
                            <goal>package</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
```

### Expected behavior

The `azure-functions-maven-plugin` should support slots similar to the `azure-webapp-maven-plugin`

### Actual behavior

The plugin does not support slots.

### Steps to reproduce the problem

Add slots to plugin.
```
<deploymentSlot>
    <name>slot</name>
    <configurationSource>test</configurationSource>
</deploymentSlot>
```

**Comment by sooji2573** [comment]
i want azure-functions-maven-plugin supporting slots

**Comment by sikang3973** [comment]
해당 기능을 제공하지 않아, 프로젝트를 운영하는데 어려움이 있습니다. 
최대한 빨리 해당 기능을 지원하기를 요청드립니다.

**Comment by sikang3973** [comment]
We are having trouble getting the project going because we don't provide this feature. We ask you to support this feature as soon as possible.

**Comment by hyomin-jang** [comment]
Hi. I've programmed using Azure. Recently I encountered difficulties with maven-plugin. You need to enhance with this function. I hope to improve this problem. Thanks.

**Comment by sooji2573** [comment]
We would like a quick problem resolution for this matter.

**Comment by Flanker32** [comment]
@IrimieBogdan, @sikang3973, @hyomin-jang , @sooji2573 Thank you all for your support, function deployment slot is in our development plan, before the release, could you please try zip deploy with azure cli? You may get the document [here](https://docs.microsoft.com/en-us/cli/azure/functionapp/deployment/source?view=azure-cli-latest#az-functionapp-deployment-source-config-zip)

**Comment by Flanker32** [comment]
@IrimieBogdan, @sikang3973, @hyomin-jang , @sooji2573 Thanks a lot for your support, deployment slot support in maven has been released in [azure-functions-maven-plugin-1.9.0](https://github.com/microsoft/azure-maven-plugins/releases/tag/azure-functions-maven-plugin-1.9.0)
